packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4227,6 +4228,11 @@ class IndexedStack extends StatelessWidget {
   /// See [Stack.fit] for more information.
   final StackFit sizing;
 
+  /// Whether or not non-visible children can be focused.
+  ///
+  /// Defaults to true.
+  final bool maintainFocusability;
```

Comment:
Let's not expose this on `IndexedStack` for now. It can be `false` by default. It's not breaking any google tests from what I see.